Signal/src/ViewControllers/HomeView/HomeViewCell.m

```diff
@@ -95,35 +87,69 @@ - (void)commontInit
     [self.dateTimeLabel setContentHuggingHorizontalHigh];
     [self.dateTimeLabel setCompressionResistanceHorizontalHigh];
 
+    self.messageStatusView = [UIImageView new];
+    [self.messageStatusView setContentHuggingHorizontalHigh];
+    [self.messageStatusView setCompressionResistanceHorizontalHigh];
+
     self.topRowView = [[UIStackView alloc] initWithArrangedSubviews:@[
         self.nameLabel,
         self.dateTimeLabel,
+        self.messageStatusView,
     ]];
     self.topRowView.axis = UILayoutConstraintAxisHorizontal;
-    self.topRowView.alignment = UIStackViewAlignmentLastBaseline;
-    [self.payloadView addArrangedSubview:self.topRowView];
+    self.topRowView.alignment = UIStackViewAlignmentCenter;
```

Comment:
Although these will look *very* close, we want the Name, Date, unread indicator label to be aligned by baseline, not vertical center.

Reply:
Done.